zetaclient/evm/evm_client.go

```diff
@@ -617,15 +611,15 @@ func (ob *EVMChainClient) IsSendOutTxProcessed(sendHash string, nonce uint64, co
 
 // FIXME: there's a chance that a txhash in OutTxChan may not deliver when Stop() is called
 // observeOutTx periodically checks all the txhash in potential outbound txs
-func (ob *EVMChainClient) observeOutTx() {
+func (ob *ChainClient) observeOutTx() {
 	// read env variables if set
 	timeoutNonce, err := strconv.Atoi(os.Getenv("OS_TIMEOUT_NONCE"))
 	if err != nil || timeoutNonce <= 0 {
 		timeoutNonce = 100 * 3 // process up to 100 hashes
 	}
 	ob.logger.ObserveOutTx.Info().Msgf("observeOutTx: using timeoutNonce %d seconds", timeoutNonce)
 
-	ticker, err := NewDynamicTicker(fmt.Sprintf("EVM_observeOutTx_%d", ob.chain.ChainId), ob.GetChainParams().OutTxTicker)
+	ticker, err := bitcoin.NewDynamicTicker(fmt.Sprintf("EVM_observeOutTx_%d", ob.chain.ChainId), ob.GetChainParams().OutTxTicker)
```

Comment:
Do you want moving `NewDynamicTicker` out of bitcoin package as it's used by both EVM/Bitcoin? Maybe we can do it in future PR.